docs/maps/index.asciidoc

```diff
@@ -9,21 +9,33 @@
 [partintro]
 --
 
-Maps enables you to parse through your geographical data at scale, with speed, and in real time. With features like multiple layers and indices in a map, plotting of raw documents, dynamic client-side styling, and global search across multiple layers, you can understand and monitor your data with ease.
-
-With Maps, you can:
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js">
+</script>
+<img
+style="width: 100%; margin: auto; display: block;"
+class="vidyard-player-embed"
+src="https://play.vidyard.com/BYzRDtH4u7RSD8wKhuEW1b.jpg"
+data-uuid="BYzRDtH4u7RSD8wKhuEW1b"
+data-v="4"
+data-type="inline"
+/>
+</br>
+++++
+
+Create beautiful maps from your geographical data. With Maps, you can:
 
 * Create maps with multiple layers and indices.
 * Upload GeoJSON files into Elasticsearch.
 * Embed your map in dashboards.
 * Symbolize features using data values.
-* Focus in on just the data you want.
+* Focus on only the data that’s important to you.
 
-*Ready to get started?* Start your tour of Maps with the <<maps-getting-started, getting started tutorial>>.
+*Ready to get started?* Start your tour of Maps with the <<maps-getting-started, getting started tutorial>> or watch the https://videos.elastic.co/watch/BYzRDtH4u7RSD8wKhuEW1b[getting started video].
 
 [float]
 === Create maps with multiple layers and indices
-You can use multiple layers and indices to show all your data in a single map. This enables your map to show how data sits relative to physical features like weather patterns, human-made features like international borders, and business-specific features like sales regions. You can plot individual documents or use aggregations to plot any data set, no matter how large.
+Use multiple layers and indices to show all your data in a single map. Show how data sits relative to physical features like weather patterns, human-made features like international borders, and business-specific features like sales regions. Plot individual documents or use aggregations to plot any data set, no matter how large.
```

Comment:
The is ok, but I feel that this makes the text feel a little "sharp".

Reply:
I have reverted these changes to keep the original text